src/WaterLily.jl

```diff
@@ -105,19 +105,23 @@ end
 
 export Simulation,sim_step!,sim_time,measure!
 
-# WriteVTKExt functions to be extended
-function vtkWriter end
-function write! end
+# WriteVTKExt export
+include("../ext/WaterLilyDefaults.jl")
```

Comment:
I am not a huge fan of this. I think we can have all in here. Otherwise following this methodology, we will need to add at least 2 files in `ext/` every time we set up a new package extension.

Reply:
I was thinking of having all the default empty functions in `ext/WaterLilyDefault.jl` and only adding a single new file if we add a new extension. But I will revert to the previous state; we can always do that later.